lib/language_pack/base.rb

```diff
@@ -93,7 +93,7 @@ def compile
         puts @deprecations.join("\n")
       end
     end
-    mcount "ruby.success"
+    mcount "success"
```

Comment:
I've been meaning to do this and compare the numbers against cytokine's librato success metric